pandas/core/groupby/groupby.py

```diff
@@ -1330,7 +1330,9 @@ def any(self, skipna: bool = True):
 
         Returns
         -------
-        bool
+        Series or DataFrame
+            DataFrame or Series of boolean values, which is True if any element is True
+            within each group, False otherwise.
```

Comment:
I've pushed the third draft for now. But I'm willing to modify these descriptions if requested.